### ISSUE TITLE
Doesn't Recognize JavaScript in HTML

### ISSUE THREAD

**Issue by renomureza** [timlrx/rehype-prism-plus#9]
Hi, this is an awesome plugin.

But I found an issue, this plugin can't recognize JavaScript in HTML.

**Code:**

```
`` `html
<script>
  console.log('Hello World');
  const hi = 'hi';
</script>
` ``
```
**Result:**

![image](https://user-images.githubusercontent.com/49445216/130358080-8ad30263-7f82-4cb4-a4b4-ab63f7d2e859.png)

**Expected behavior:**

Should be the same as the [@mapbox/rehype-prism](https://github.com/mapbox/rehype-prism) plugin, it's recognized. Like this:

![image](https://user-images.githubusercontent.com/49445216/130358469-4932bec3-f557-493b-a860-dae7a2ba3558.png)

**Comment by timlrx** [comment]
Good catch, thanks! The bug is caused because I am passing each line through `refactor` rather than parsing it as an entire code block. Let me try to fix it

**Comment by renomureza** [comment]
This may also be related, multiline comments are not handled properly. 

**rehype-prism-plus**

![image](https://user-images.githubusercontent.com/49445216/130374967-81342936-4cda-4fe5-9f73-1324ce196dae.png)

**@mapbox/rehype-prism**

![image](https://user-images.githubusercontent.com/49445216/130375038-fde097ef-0309-49dd-add3-546a2e4a5cf3.png)

**Comment by timlrx** [comment]
Yes, that's related

**Comment by timlrx** [comment]
I pushed a fix in #10. Let me know if that works for you?

**Comment by renomureza** [comment]
Yes it works, as expected 🥳

![image](https://user-images.githubusercontent.com/49445216/130554313-373dfa23-9fb2-4914-86a0-3b5d064283b6.png)

Waiting for release.

**Comment by timlrx** [comment]
Published as v0.0.6!
